docs/content/developer/getting-started/simple-token-transfer.mdx

```diff
@@ -0,0 +1,438 @@
+    participant Contract
+    participant Treasury
+    participant Recipient
+
+    Sender->>Contract: init(witness, ctx)
+    Contract->>Treasury: create_currency(witness)
+    Treasury-->>Contract: Returns TreasuryCap, Metadata
+    Contract->>Sender: transfer(treasury, sender)
+
+    Sender->>Contract: mint(treasury_cap, amount, recipient, ctx)
+    Contract->>Treasury: mint(treasury_cap, amount)
+    Treasury-->>Contract: Returns Coin
+    Contract->>Recipient: transfer(coin, recipient)
+
+    Sender->>Contract: transfer(token, amount, recipient, ctx)
+    Contract->>Treasury: split(token, amount)
+    Treasury-->>Contract: Returns CoinToTransfer
+    Contract->>Recipient: transfer(CoinToTransfer, recipient)
+
+    Sender->>Contract: balances(token)
+    Contract-->>Sender: Returns token.balance
+```
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/) >=v20.18.0
+- [npx](https://www.npmjs.com/package/npx) >=10.8.2
+- [iota](https://github.com/iotaledger/iota/releases) >=0.7.3 
+
+## Create a Move Package
+
+Run the following command to create a Move package
+
+```bash
+iota move new token_transfer
+```
+
+This will generate a new Move package in a folder named token-transfer.
+
+## Configure the IOTA CLI
+
+To run the `mint` function from the frontend app, you must be the owner of the package; otherwise, you cannot execute the function. To become the owner, you need to publish the package, and this process can only be done through the CLI. Note that the CLI and browser wallet extension use separate accounts. In the CLI, you can update the active address and private key to match the account used in the browser wallet extension. By aligning these accounts, you ensure the same account is used in both the CLI and the wallet extension, enabling seamless transactions in your frontend app.
+
+Before publishing the package, you need to configure your IOTA CLI with your wallet details.
+
+### Navigate to the IOTA configuration directory
+
+```bash
+cd ~/.iota/iota_config/
+```
+
+### Update the Private Key and Active Address
+
+`iota.keystore`: Replace the existing private key with your IOTA wallet private key.
+
+`client.yaml`: Replace the `active_address` field with your IOTA wallet address.
+
+### Package Overview
+
+#### [`init`](https://github.com/iota-community/token-transfer-tutorial/blob/ea71112c1156d1885567353fee0a03e09308e293/sources/token_transfer.move#L9-L13)
+
+- Executes at the time of publication of the package.
+- Takes `MY_TOKEN` as a witness and the transaction context (ctx).
+- Uses the `coin::create_currency` function to create the token by specifying the name, symbol, and decimal precision of the token.
+- The `coin::create_currency` function returns a TreasuryCap, which allows the owner to mint and transfer the token, and the metadata of the token.
+- Transfers the metadata to the owner of the contract using `transfer::public_freeze_object(metadata)` and `transfer::public_transfer(treasury, ctx.sender())`.
```

Comment:
Please add links to each ref, https://docs.iota.org/references/framework/iota-framework/transfer#0x2_transfer_public_freeze_object, https://docs.iota.org/references/framework/iota-framework/transfer#function-public_transfer, https://docs.iota.org/references/framework/iota-framework/tx_context#0x2_tx_context_sender

Reply:
Sure.